stl/inc/chrono

```diff
@@ -608,41 +607,8 @@ namespace chrono {
 #pragma warning(disable : 28112) // A variable which is accessed via an Interlocked function must
                                  // always be accessed via an Interlocked function.

```

Comment:
`#pragma warning push/disable/pop` was added for the `_InterlockedMeow` calls. This revert is removing the `pop` but not the `push/disable`.